multiply.h

```diff
@@ -66,6 +66,20 @@ static inline float MaxFloat32(__m128 a) {
   return *reinterpret_cast<float*>(&a);
 }
 
+static inline __m128i UnquantNop(__m128i arg) {
```

Comment:
This isn't test related and should probably be separate.  Also, I don't see who would call this function?  You'll need a Nop functor for WriteC, but it can just have an empty body.